app/assets/stylesheets/modules/_family.scss

```diff
@@ -0,0 +1,149 @@
+.family {
+  // @include span-columns(12 of 12);
+  width: 95%;
+  margin-left: 2.5%;
+
+  background-color: $white;
+  border-radius: $base-border-radius;
+  padding: em(40);
+
+  @include media($medium-screen-up) {
+    @include span-columns(10 of 12);
+    @include shift(1);
+
+    padding: em(90);
+  }
+
+  &-section-one {
+
+    &-content {
+      @include span-columns(10 0f 10);
+
+      @include media($medium-screen-up) {
+        @include span-columns(6.8 of 10);
+        margin-left: em(10);
+      }
+    }
+
+    &-image {
+      @include span-columns(12 of 12);
+      background: image-url("julian-michelle.png");
+      background-repeat: no-repeat;
+      background-size: 100%;
+      height: em(200);
+      display: block;
+
+      @include media($medium-screen-up) {
+        @include span-columns(3 of 10);
+        background-size: 100%;
+        display: inline-block;
+        height: em(120);
+      }
+
+    }
+
+  }
+
+  &-section-two-image {
+    background: image-url("sf-move.png");
+    background-repeat: no-repeat;
+    background-size: em(260);
+    display: block;
+    height: em(110);
+    width: em(260);
+    margin: 0 auto;
+
+    @include media($medium-screen-up) {
+      background-size: em(460);
+      height: em(162);
+      width: em(460);
+      margin-top: em(60);
+      margin-bottom: em(80);
+    }
+  }
+
+  &-content {
+
+    &-text {
+      @include span-columns(12 of 12);
+
+      font-weight: 300;
+      padding-bottom: $small-spacing/2;
+      font-size: em(18);
+      line-height: em(30);
+
+      &-secondary {
+        @include span-columns(8 of 12);
+        font-weight: 300;
+        padding-bottom: $small-spacing/2;
+      }
+    }
+
+    &-cta {
+      background-color: $action-color;
+      border-bottom: em(3) solid $dark-pink;
+      border-radius: $base-border-radius;
+      display: inline-block;
+      font-weight: 500;
+      margin-top: em(10);
+      margin-right: em(10);
+      padding: em(5) em(10);
+
+      a {
+        color: $white;
+      }
+
+      &:hover {
+        background-color: tint($action-color, 20%);
+        border-bottom: em(3) solid tint($dark-pink, 20%);
+        cursor: pointer;
+      }
+    }
+  }
+
+  &-section-two {
+    @include span-columns(12);
+  }
+
+}
+
+.highlight {
+  display: inline-block;
+  font-weight: 500;
+
+  &-color {
+    color: $base-accent-color;
+  }
+
+  &-centered {
+    text-align: center;
+  }
+}
+
+.michelle {
+  display: inline-block;
+  height: em(25);
+  width: em(30);
+}
+
+.julian {
+  display: inline-block;
+  height: em(27);
+  width: em(30);
+}
+
+.family-section-three-image {
+  @include span-columns(10 of 10);
+  background: image-url("faces.png");
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height: em(300);
```

Comment:
Properties should be ordered background, background-repeat, background-size, display, height